src/lsm/lsm_stat.c

```diff
@@ -39,7 +39,7 @@ __curstat_lsm_init(
 	WT_ERR(__wt_scr_alloc(session, 0, &uribuf));
 
 	/* Propagate all, fast and/or clear to the cursors we open. */
-	if (!F_ISSET(cst, WT_CONN_STAT_NONE)) {
+	if (cst != 0) {
```

Comment:
It looks to me like `cst` can't be NULL here, so adding this test is almost certainly going to trigger a Coverity complaint, Coverity doesn't like it if you check a pointer against NULL, but don't check it against NULL everywhere.